src/common/components/FriendItem/index.js

```diff
@@ -13,13 +16,23 @@ function FriendItem({
 }) {
   const [isHovered, setIsHovered] = useState(false);
   const statusIconPropsStyle = isHovered ? 'var(--gray-12)' : 'var(--gray-2)';
+  const dispatch = useDispatch();
+
+  const friendItemEnter = () => {
+    setIsHovered(true);
+    dispatch(savingActiveItem(activeItem));
+  };
+  const friendItemLeave = () => {
+    setIsHovered(false);
+    dispatch(savingActiveItem(''));
+  };
```

Comment:
Setting active item can be done before click, don't need to use hover there as current action will execute multiple times with no reason